### PR TITLE
Update docs theme 1.1

### PR DESCRIPTION
**Pull request by dgarcia360**
Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/133

ScyllaDB Sphinx Theme 1.1 is now released 🥳 

We’ve made a number of updates to update all our dependencies to the latest version and introduced new directives you can use to write great docs.

You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/upgrade/CHANGELOG.html#january-2022).

## ⚠️Breaking changes

Before,  the theme installed [poetry 1.1.x](https://python-poetry.org/) as a dependency to manage Python dependencies. However, ``poetry 1.2.x`` changed the installation method. Therefore, we've decided to [#307 Make poetry a prerequisite](https://github.com/scylladb/sphinx-scylladb-theme/issues/307) so that you can decide to install the poetry version you prefer.

To preview the docs locally, you should uninstall the previous version of poetry. Then, install the latest version:

1. Uninstall Poetry 1.1.x.

```
curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_UNINSTALL=1 python -
```

2. Install Poetry 1.2.x. For detailed instructions, see [Poetry installation](https://python-poetry.org/docs/master/#installation).

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Uninstall poetry 1.1 and install poetry 1.2. For more information, see **Breaking changes** notice above.

3. Enter the docs folder, and run:

```
make preview
````

4. Open http://127.0.0.1:5500/ with your favorite browser. The doc should render without errors, and the version should be Sphinx Theme version (see the footer) must be ``1.1.x``:

image

### CODE REVIEW

**Comment by dgarcia360** [comment]
@tzach Please review

File: docs/conf.py [hunk 1]
```diff
@@ -3,9 +3,6 @@
 import sys
 from datetime import date
 
-import recommonmark
```
Comment (tzach) [inline docs/conf.py:6]: dont we need recommonmark to parse MD?

File: docs/conf.py [hunk 2]
```diff
@@ -76,7 +56,6 @@ def setup(sphinx):
     'github_issues_repository': 'scylladb/care-pet',
     'github_repository': 'scylladb/care-pet',
     'hide_edit_this_page_button': 'false',
-    'hide_sidebar_index': 'true',
```
Comment (tzach) [inline docs/conf.py:79]: why removing this?

File: docs/index.rst [hunk 3]
```diff
@@ -76,7 +78,7 @@
 
 .. raw:: html
```
Comment (tzach) [inline docs/index.rst:79]: do we still need HTML in the page?

**Review by tzach — COMMENTED** [review]
few questions

**Comment by tzach** [comment]
failed to build the docs
```
✔ care-pet/docs [docs-theme-1-1|✔] 
09:03 $ make preview 
/home/tzach/.local/bin/poetry install
make: /home/tzach/.local/bin/poetry: No such file or directory
make: *** [Makefile:30: setup] Error 127
```

File: .github/workflows/docs-pr@v2.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name: "Docs / Build PR"
```
[the remark this answers is not in the document]
Comment (dgarcia360) [reply .github/workflows/docs-pr@v2.yaml:1]: Added a comment "# For more information, see https://sphinx-theme.scylladb.com/master/github-pages.html#available-workflows"

File: docs/index.rst (hunk 3 above)
Comment (dgarcia360) [reply docs/index.rst:79]: Yes, we managed to remove one html tag but the issue is still open: https://github.com/scylladb/sphinx-scylladb-theme/issues/217

File: docs/conf.py (hunk 1 above)
Comment (dgarcia360) [reply docs/conf.py:6]: Recommonmark it's still installed, but we don't need to do this custom setup anymore:

image

Now, only need to list it as a extension:

image

File: docs/conf.py (hunk 2 above)
Comment (dgarcia360) [reply docs/conf.py:79]: See https://github.com/scylladb/sphinx-scylladb-theme/issues/229

**Comment by dgarcia360** [comment]
> failed to build the docs

Please see [Breaking changes](https://github.com/scylladb/care-pet/pull/56#issue-1121573937).

File: .github/workflows/docs-pages@v3.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name: "Docs / Publish"
```
[the remark this answers is not in the document]
Comment (dgarcia360) [reply .github/workflows/docs-pages@v3.yaml:1]: These lines are new:

https://github.com/scylladb/sphinx-scylladb-theme/commit/042e6abcf10d5a7768b8f61db65f67e572a7396f#diff-29776b6a4a2ef6dcc7d24f25ebe1869eb05f8b12831a5f69b43590f1e083e8e6

**Pull request merged**